src/lib/core/cxx_abi.h

```diff
@@ -21,12 +21,6 @@ extern "C" {
  */
 const char *
 cxx_abi_demangle(const char *mangled_name);
-
-/*
- * Free resources used by C++ ABI.
- */
-void
-cxx_abi_cleanup(void *unused);
```

Comment:
This should be in the previous patch. Please rebase. Move of `ENABLE_BACKTRACE` too.

Reply:
I beg your pardon, Friday evening...

Fixed: moved these diffs to C++ ABI refactoring patch.